test/membrane/binarydata.spec.js

```diff
@@ -1,28 +1,30 @@
 import createVirtualEnvironment from '@locker/near-membrane-dom';
 
 describe('TypedArray', () => {
-    describe('Atomics', () => {
-        it('operates on atomic-friendly typed arrays', (done) => {
-            const env = createVirtualEnvironment(window, window, {
-                endowments: Object.getOwnPropertyDescriptors({ done, expect }),
-            });
+    if (typeof Atomics !== 'undefined') {
```

Comment:
This is disabled in Safari TP, so we should just skip it if the API isn't available.

Reply:
Interesting are Atomics on their way out again due to security or something?

Reply:
No, they're not. It looks like they are disabled in Safari as well